server/controllers/socialAuthController.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * @description This takes in the HTTP server request,
+ performs the logic and returns the server response with the user object.
+ * @param  {HttpRequest} req
+ * @param  {HttResponse} res
+ * @returns {HttpResponse} Server Response
+ */
+const socialAuthController = (req, res) => {
```

Comment:
call this `socialAuth`. It is already in the controllers folder so you don't need to specify again that it is a controller